src/Symfony/Component/Console/Tester/CommandTester.php

```diff
@@ -47,8 +47,8 @@ public function __construct(Command $command)
      *  * decorated:   Sets the output decorated flag
      *  * verbosity:   Sets the output verbosity flag
      *
-     * @param array $input   An array of arguments and options
-     * @param array $options An array of options
+     * @param array $input   An array of command arguments and options
+     * @param array $options An array of execution options
```

Comment:
you should probably update the description above to say ``Available execution options:`` then